server/e2e/common.go

```
@@ -244,25 +247,42 @@
+	return assert.Regexp(
+		t,
+		regexp.MustCompile(strings.ReplaceAll(aligningJSON(t, expected), "[", "\\[")),
+		aligningJSON(t, actual),
+	)
+}
 
-	var rx1 map[string]interface{}
-	err := json.Unmarshal([]byte(rx), &rx1)
-	assert.Nil(t, err)
+func RegexpJSONEReadCloser(t *testing.T, actual io.ReadCloser, expected string) bool {
+	defer actual.Close()
```

golangci-lint (1.62.2)

Error return value of `actual.Close` is not checked

(errcheck)